Glyssen/Block.cs

```diff
@@ -545,18 +545,41 @@ public string GetTextAsHtml(bool showVerseNumbers, bool rightToLeftScript)
 			return bldr.ToString();
 		}
 
+		public bool TryGetLeadingPunctuation(out ScriptText scriptTextWithLeadingPunctuation)
+		{
```

Comment:
Given this name, I think I would expect it to return the text itself (i.e., return initialScriptText.Content) rather than the ScriptText.